### PR TITLE
enforce: add init enforce on QPS

### LINKED ISSUE #85: support QPS enforcement of each client

### PR DESCRIPTION
**Pull request by yichengq**
fixes #85

### CODE REVIEW

**Comment by yunxing** [comment]
LGTM

File: server/main.go [hunk 1]
```diff
@@ -51,6 +52,9 @@ func main() {
 		}
 	}
 
+	// 0x1234 is the client ID for cfsctl, and its quota is 10 req/sec.
+	enforce.SetQuota(0x1234, 10)
```
Comment (xiang90) [inline server/main.go:56]: hmm... what is this for?

File: enforce/enforce.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+var (
+	mu sync.Mutex
+	// the number of client operations in the last checkpoint
+	ops    = make(map[int64]int)
+	quotas = make(map[int64]int)
+)
+
+func init() {
+	updateOps()
```
Comment (xiang90) [inline enforce/enforce.go:22]: why do we need to do this in init() ?

File: enforce/enforce.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+		n, err := fmt.Sscanf(k, "%x_ops", &id)
+		if err != nil || n != 1 {
+			continue
+		}
+		ops[id] = int(v)
+	}
+}
+
+func HasQuota(clientID int64) bool {
+	if _, ok := quotas[clientID]; !ok {
```
Comment (xiang90) [inline enforce/enforce.go:46]: why do not need to hold the lock?

**Pull request merged**